src/rust/ide/lib/view/src/graph_editor.rs

```diff
@@ -1187,6 +1213,15 @@ impl GraphEditorModel {
         }
     }
 
+    fn set_documentation_visibility(&self, is_vis:bool) {
+        if is_vis { self.app.remove_child(&self.doc_view) }
+        else      { self.app.add_child(&self.doc_view)    }
+    }
+
+    fn is_doc_visible(&self) -> bool {
+        self.doc_view.has_parent()
+    }
```

Comment:
the whole pr is full of inconsistent names. Sometimes you sue `doc`, sometimes `documentation`. Please make it consistent. I'll vote for `doc_view` instead.

Reply:
done